lib/cli/configuration.js

```diff
@@ -325,3 +326,21 @@ exports.load = (file) => {
     throw error;
   }
 };
+
+/**
+ * @summary Returns defualt configuration file path
+ * @function
+ * @public
+ *
+ * @param {array} fallbacks - file paths for files that may or may not exist
+ * @returns {string} file path
+ *
+ * @example
+ * const configPath = configuration.getDefault([ './foo.js', './bar.js' ]);
+ * console.log(configPath);
+ */
+exports.getDefaultPath = (fallbacks) => {
```

Comment:
To me *how* we get the default path might change, this way the names always approps. But no strong feels.